python/lsst/ts/scriptqueue/ui/script_queue_commander.py

```diff
@@ -127,15 +154,60 @@ async def do_add(self, args):
             raise ValueError("Need at least 2 arguments")
         is_standard = self.get_is_standard(args[0])
         path = args[1]
-        if len(args) == 2:
-            config_yaml = ""
-        elif len(args) == 3 and args[2].startswith("@"):
-            config_path = args[2][1:]
+
+        # Parse config
+        config_args = []
+        config_path = None
+        config_yaml_items = []
+        config_start_ind = 2
+        options_start_ind = 2
+        if len(args) > 2 and args[2].startswith("@"):
+            config_path = args[2]
```

Comment:
I think you need to strip the "`@`" of the name, otherwise it won't find the file unless it also has the "@" on it. 

I tried...

Reply:
Good catch. I fixed that and also add a default suffix of .yaml (and documented that in the help string). Thank you.